addons/stock_account/models/stock_move.py

```diff
@@ -66,6 +66,37 @@ def _get_valued_types(self):
         """
         return ['in', 'out', 'dropshipped', 'dropshipped_returned']
 
+    def _filter_in_out(self):
+        in_moves, out_moves = self._get_in_out_moves()
+        is_move_in_ids = set()
+        is_move_out_ids = set()
```

Comment:
You can drop the `is_` from the variable names. `move_in_ids` is easier to understand.